src/features/collaborator-avatars/CollaboratorList.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/variables';
+
+.collaborator-list {
+    border: solid 1px $seesee;
+    border-radius: 4px;
+    box-shadow: 0 2px 3px 0 rgba(0, 0, 0, .08) inset;
+    height: 210px;
+    margin: 0;
+    min-width: 400px;
+    overflow-y: scroll;
+
+    .collaborator-list-item {
```

Comment:
This should ideally be part of its own scss imported within its parent component

Reply:
I split this into a second file. Let me know if I did that correctly. Why the parent instead of the component the styles are for?

Reply:
Oh I meant the parent component for the styles which in this case would be CollaboratorListItem

Reply:
Gotcha. I'll move that to the correct component.